appservice/src/SiteWrapper.ts

```diff
@@ -116,6 +120,67 @@ export class SiteWrapper {
         this.log(outputChannel, 'Deployment completed.');
     }
 
+    public async localGitDeploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, servicePlan: string): Promise<void> {
+        vscode.window.showInformationMessage('It updated to 2.2');
+        let taskResults: [User, SiteConfigResource];
+        const kuduClient: KuduClient = await this.getKuduClient(client);
+        const yes: string = 'Yes';
+        const pushReject: string = 'Push rejected due to Git history diverging. Force push?';
+
+        if (!this.slotName) {
```

Comment:
You can simplify this code quite a bit (and completely get rid of the taskResults variable)
```
const [publishCredentials, config]: [User, SiteConfigResource] = await Promise.all([
    this.getWebAppPublishCredential(client),
    this.getSiteConfig(client)
]);
```

NOTE: You'll have to change the return type of `this.getSiteConfig` to `SiteConfigResource` - looks like it's incorrect at the moment? Not sure what the difference between `SiteConfig` and `SiteConfigResource` is